packages/api/src/command/medical/docref-mapping/get-docref-mapping.ts

```diff
@@ -9,11 +9,21 @@ export const getDocRefMapping = async (id: string): Promise<DocRefMapping | unde
   return docRef ?? undefined;
 };
 
-export const getAllDocRefMappingByRequestId = async (
-  requestId: string
-): Promise<DocRefMapping[]> => {
+export const getAllDocRefMapping = async ({
+  requestId,
+  patientId,
+  cxId,
+}: {
+  requestId?: string;
+  patientId?: string;
+  cxId?: string;
+}): Promise<DocRefMapping[]> => {
   const docRefs = await DocRefMappingModel.findAll({
-    where: { requestId },
+    where: {
+      requestId,
+      ...(patientId && { patientId }),
+      ...(cxId && { cxId }),
```

Comment:
They can all be empty, so I suggest we check if at least one is set. Also, if `requestId` can be empty, I suggest we use the same code that we have for patient and cx id, where we only add it if present.